### DISCUSSION TITLE
Calculation

### DISCUSSION THREAD

**Discussion by kyronjavier** [taydz/Tafe230213#7] [General]
Hi I found myself confused with the calculation for the Currency Calculator. Do you know what's the conversion for Indian Rupee to US Dollar, Euro, and British Pound?

**Comment by taydz** [comment]
Hey, yes I can help you with that!
₹ 1 Indian Rupee = $0.011492628 US Dollar, €0.013492774 Euro, and £0.0098339397 British Pound.
